### ISSUE TITLE
Text field moves cursor to start of text when clicking in padding area

### ISSUE THREAD

**Issue by surma** [breadboard-ai/breadboard#3631]
If you click into the area between the text and the border of a text field, the cursor is set to the start of the text, rather than the end.

![Screenshot 2024-10-27 at 09 29 57](https://github.com/user-attachments/assets/7b731eed-9878-4457-8080-361b7849eb14)

**Comment by paullewis** [comment]
Yeah, this one is a common issue with inputs, and apparently it varies based on OS and browser. Short of adding some JS which would force the caret position (which I'm disinclined to do because it feels like it's messing with default behaviors), there doesn't seem to be a suitable fix.

We could change the `line-height` and the `text-indent` values, but that would make the leading between lines very odd.
